contrib/hawq_package_conf/apache_hawq.spec

```diff
@@ -0,0 +1,81 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+%global    _enable_debug_package  0
+%global    debug_package          %{nil}
+%global    __os_install_post      /usr/lib/rpm/brp-compress %{nil}
+%define    hawq_version           %{_hawq_version}
+%define    build_number           %{_build_number}
+%define    rpm_os_version         %{_rpm_os_version}
+%define    arch                   x86_64
+%define    installdir             /usr/local/%{name}
+
+Name:       apache-hawq
+Summary:    Hadoop Native SQL powered by Apache HAWQ (incubating)
+Version:    %{hawq_version}
+Release:    %{build_number}.%{rpm_os_version}
+License:    ASL 2.0
+Group:      Applications/Databases
+URL:        http://hawq.incubator.apache.org
+Prefix:     /usr/local
+BuildArch:  %{arch}
+SOURCE0 :   %{name}-%{hawq_version}-bin.tar.gz
+Requires:   libgsasl, krb5-libs, libicu, protobuf >= 2.5.0, json-c >= 0.9, net-snmp-libs, thrift >= 0.9.1, boost >= 1.53.0
+%if %{rpm_os_version} == el6
+Requires: openssl
+%else
+Requires: openssl-libs
+%endif
+Requires(pre): shadow-utils
+
+AutoReqProv:    no
+
+%description
+%{summary}
```

Comment:
It would be better to provide a real description than rather using the summary.  How about something like:

```
Apache HAWQ (incubating) combines exceptional MPP-based analytics
performance, robust ANSI SQL compliance, Hadoop ecosystem
integration and manageability, and flexible data-store format
support, all natively in Hadoop, no connectors required.

Built from a decade’s worth of massively parallel
processing (MPP) expertise developed through the creation of open
source Greenplum® Database and PostgreSQL, HAWQ enables you to
swiftly and interactively query Hadoop data, natively via HDFS.
```